colearn_keras/keras_mnist.py

```diff
@@ -39,21 +39,21 @@
 
 # The dataloader needs to be registered before the models that reference it
 @FactoryRegistry.register_dataloader("KERAS_MNIST")
-def prepare_data_loaders(train_folder: str,
+def prepare_data_loaders(location: str,
                          train_ratio: float = 0.9,
                          batch_size: int = 32,
-                         **_kwargs) -> Tuple[PrefetchDataset, PrefetchDataset]:
+                         ) -> Tuple[PrefetchDataset, PrefetchDataset]:
     """
     Load training data from folders and create train and test dataloader
 
-    :param train_folder: Path to training dataset
+    :param location: Path to training dataset
     :param train_ratio: What portion of train_data should be used as test set
     :param batch_size:
     :param _kwargs: Residual parameters not used by this function
```

Comment:
_kwargs is removed from the function, the doc should be updated as well

Reply:
Well spotted!